### PR TITLE
Add navbar to website

### PR DESCRIPTION
**Pull request by joshuagcamacho**

### CODE REVIEW

File: pages/components/header.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import Head from "next/head"
+import NavBar from "./navbar"
```
Comment (stevenning203) [inline pages/components/header.tsx:3]: theres no jsdoc annotation here. one more violation an ur firied.

**Review by stevenning203 — COMMENTED** [review]
garbage code! you should be ashamed

**Comment by joshuagcamacho** [comment]
Sorry sire!

**Comment by stevenning203** [comment]
It's ok, just be careful.

**Pull request merged**